### PR TITLE
[RA-1 Ch06] 6.3.5 Image Security

### PR DESCRIPTION
**Pull request by pgoyal01**
Add "6.3.5 Image Security" Content

### CODE REVIEW

**Comment by pgoyal01** [comment]
Fixes #1395

File: doc/ref_arch/openstack/chapters/chapter06.md [hunk 1]
```diff
@@ -202,6 +202,12 @@ Operators typically do not implement Security Groups when ussing SR-IOV or DPDK
 <a name="6.3.5"></a>
 ### 6.3.5 Image Security
 
+OpenStack provides valuabele guidance on trusted image creation process, and image signature verification (please see "Trusted Images" section in the [OpenStack Security Guide](https://docs.openstack.org/security-guide/instance-management/security-services-for-instances.html#trusted-images/) documentation). Adding Signed Images to the Image Service (Glance) is specified in [OpenStack Operations Guide](https://docs.openstack.org/operations-guide/ops-user-facing-operations.html#adding-signed-images).
+
+Integrity Verification at instantiation is required by [ETSI NFV SEC021](https://portal.etsi.org/webapp/WorkProgram/Report_WorkItem.asp?WKI_ID=53601) and the creation of signature per individual artifact in the VNF package is required by [ETSI NFV SOL004](http://www.etsi.org/deliver/etsi_gs/NFV-SOL/001_099/004/02.03.01_60/gs_nfv-sol004v020301p.pdf).
```
Comment (sukhdevkapur) [inline doc/ref_arch/openstack/chapters/chapter06.md:207]: Minor typo - Verification at the time of instantiation

Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter06.md:207]: @sukhdevkapur Corrected -- Thanks

**Comment by yeelinglam** [comment]
@pgoyal01 - Please to add the following, thanks!

1. Scanning / certification process to ingest the images
2. Enterprise required toolset, e.g. approved anti-virus/malware detection, IAM

**Comment by pgoyal01** [comment]
@yeelinglam incorporated except for IAM as that is required everywhere

File: doc/ref_arch/openstack/chapters/chapter06.md [hunk 2]
```diff
@@ -202,6 +202,14 @@ Operators typically do not implement Security Groups when ussing SR-IOV or DPDK
 <a name="6.3.5"></a>
 ### 6.3.5 Image Security
 
+OpenStack provides valuable guidance on trusted image creation process, and image signature verification (please see "Trusted Images" section in the [OpenStack Security Guide](https://docs.openstack.org/security-guide/instance-management/security-services-for-instances.html#trusted-images/) documentation). The OpenStack Security Guide includes reference to the "[OpenStack Virtual Machine Image Guide](https://docs.openstack.org/image-guide/) that "describes how to obtain, create, and modify" OpenStack compatible virtual machine images.
```
Comment (iangardner22) [inline doc/ref_arch/openstack/chapters/chapter06.md:205]: Should this say "docs.openstack.org provides valuable..."

**Comment by pgoyal01** [comment]
@rabi-abdel Can you please approve and merge. Thanks

**Pull request merged**